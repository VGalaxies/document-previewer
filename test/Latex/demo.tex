\documentclass[UTF8]{ctexart}

\title{文档预览器}
\author{刘添翼}
\date{\today}

\usepackage{indentfirst}
\setlength{\parindent}{2em}

\usepackage{hyperref}
\hypersetup{
	colorlinks=true,
}

\usepackage{listings}
\lstset{
	columns=fixed,       
	numbers=left,     % 在左侧显示行号
	language=java,    % 设置语言
}

\usepackage{graphicx}
\usepackage{float}

\begin{document}
\maketitle
\tableofcontents
\section{最后期限}
Project2的最后期限是\textbf{北京时间2021年11月14日23时59分}。


在此之前，每位同学需要提交课程项目的源码、可执行文件和项目报告到...

\section{技术指标}

\subsection{对Microsoft Word文档的支持}
借助已经存在的开源的\emph{C++}库...

\paragraph{问题2.1.}
我对\underline{Microsoft Word}和XML都完全不熟悉，我要怎么办？

这里有一个超链接\href{https://cn.bing.com/}{必应}，STFW。

\begin{itemize}
    \item Fedora
    \item Fedora Spin
    \item Fedora Silverblue
\end{itemize}

\begin{enumerate}
    \item Fedora CoreOS
    \item Fedora Silverblue
    \item Fedora Spin
\end{enumerate}

\begin{lstlisting}
public class Demo {
    public static int binaryOne(int n) {
        if (n <= 0)
            throw new RuntimeException();

        if (n == 1) {
            return 1;
        } else if (n % 2 == 0) {
            return binaryOne(n / 2);
        } else {
            return 1 + binaryOne(n / 2);
        }
    }


    public static void main(String[] args) {
        binaryOne(3);
    }
}
\end{lstlisting}

\begin{figure}[H]
	\centering
	\includegraphics{img/88833228_p0.jpg}
\end{figure}

\begin{equation}
J(x)=L i(x)-\sum_{\rho} L i\left(x^{\rho}\right)-\ln 2+\int_{0}^{\infty} \frac{d t}{t\left(t^{2}-1\right) \ln t}
\end{equation}

\end{document}