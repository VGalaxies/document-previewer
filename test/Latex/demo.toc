\contentsline {section}{\numberline {1}最后期限}{1}{section.1}%
\contentsline {section}{\numberline {2}技术指标}{1}{section.2}%
\contentsline {subsection}{\numberline {2.1}对Microsoft Word文档的支持}{1}{subsection.2.1}%
\contentsline {paragraph}{问题2.1.}{1}{paragraph*.2}%
