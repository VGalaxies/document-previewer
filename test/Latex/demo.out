\BOOKMARK [1][-]{section.1}{\376\377\147\000\124\016\147\037\226\120}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\142\200\147\057\143\007\150\007}{}% 2
\BOOKMARK [2][-]{subsection.2.1}{\376\377\133\371\000M\000i\000c\000r\000o\000s\000o\000f\000t\000\040\000W\000o\000r\000d\145\207\150\143\166\204\145\057\143\001}{section.2}% 3
